Description: Enable KDE4 exceptions for libraw 0.16.1 compatibility
Author: Scott Kitterman <scott@kitterman.com>
Origin: upstream, 46898ec97a0462eb08a5b6d09999450a86b27aae
Bug-Debian: http://bugs.debian.org/788975
Last-Update: 2015-07-29

Index: libkdcraw-15.04.3/libkdcraw/CMakeLists.txt
===================================================================
--- libkdcraw-15.04.3.orig/libkdcraw/CMakeLists.txt	2015-07-29 00:38:25.322854519 -0400
+++ libkdcraw-15.04.3/libkdcraw/CMakeLists.txt	2015-07-29 00:39:06.118855053 -0400
@@ -4,6 +4,8 @@
 # Redistribution and use is allowed according to the terms of the BSD license.
 # For details see the accompanying COPYING-CMAKE-SCRIPTS file.
 
+add_definitions(${KDE4_ENABLE_EXCEPTIONS})
+
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/version.h.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/version.h)
 
 set(kdcraw_LIB_SRCS kdcraw.cpp
